deeplabcut/pose_estimation_pytorch/apis/analyze_videos.py

```diff
@@ -233,11 +251,13 @@ def analyze_videos(
         identity_only: sub-call for auto_track. If ``True`` and animal identity was
             learned by the model, assembly and tracking rely exclusively on identity
             prediction.
-        cropping: list or None, optional, default=None
-            List of cropping coordinates as [x1, x2, y1, y2].
-            Note that the same cropping parameters will then be used for all videos.
-            If different video crops are desired, run ``analyze_videos`` on individual
-            videos with the corresponding cropping coordinates.
+        cropping: List of cropping coordinates as [x1, x2, y1, y2]. Note that the same
+            cropping parameters will then be used for all videos. If different video
+            crops are desired, run ``analyze_videos`` on individual videos with the
+            corresponding cropping coordinates.
+        save_as_df: Cannot be used when `use_shelve` is True. Saves the video
+            predictions (before tracking results) to an H5 file containing a pandas
+            DataFrame.
```

Comment:
Maybe we should also say that if `save_as_csv==True` it also saves the predictions? (here or in `save_as_csv`  argument description)